uint8 local_index       # The number within the key chain
uint8 remote_addr       # The int dest addr (since the pose index is unknown)
uint16 measured_range   # The int (range, multiplied by 100)